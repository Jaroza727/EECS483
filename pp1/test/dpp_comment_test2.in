/* multiline comments
// with one line comment
within */
which ends incorrect */

/* comment
does not end
