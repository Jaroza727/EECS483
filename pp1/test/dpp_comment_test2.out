


which ends incorrect */




*** Error.
*** Input ends with unterminated comment


